apps/src/code-studio/pd/foorm/Foorm.jsx

```diff
@@ -26,15 +29,19 @@ export default class Foorm extends React.Component {
 
   onComplete = (survey, options) => {
     console.log('Survey results: ' + JSON.stringify(survey.data));
```

Comment:
nit: could probably remove this at some point.

Reply:
yep, will do